FiveCalls/FiveCalls/Dashboard.swift

```diff
@@ -25,7 +25,7 @@ struct Dashboard: View {
                             showLocationSheet.toggle()
                         }
                         .sheet(isPresented: $showLocationSheet) {
-                            LocationSheet(location: appState.location, delegate: (UIApplication.shared.delegate as! AppDelegate))
+                            LocationSheet(location: appState.location, setLocation: appState.setLocation)
```

Comment:
Instead of passing this tightly coupled completion function, would it make more sense to just include the app state as an environment object? That way, if the mechanisms change for what we do in the location sheet, we can just adjust what we call from there.

Reply:
Yeah, that does make more sense.

Reply:
resolved with proper unidirectional data flow